Samples/src/VisualDebugger/Startup.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace vis_sim
+{
+    internal class Startup
```

Comment:
At first glance, it's unclear about why there is a `Startup` and why it's not called from `Program.Main`. Adding some description here would be nice.